environments/se/core/src/test/java/org/jboss/weld/environment/se/test/builder/WeldBuilderTest.java

```diff
@@ -211,4 +211,19 @@ public void testNoBeanArchivesFound() {
         new Weld().disableDiscovery().initialize();
     }
 
+    @Test
+    public void testMultipleWeldInstancesCreated() {
```

Comment:
`WeldBuilderTest.testInitializedContainers()` is already testing multiple containers. So maybe we could just rename the original test method and add the `WeldContainer.getRunningContainerIds()` assertions?